{{ cookiecutter.repo_name }}/docs/structure/README.md

```diff
@@ -1,149 +1,124 @@
-file is based on the
-[common configuration](https://gds-way.cloudapps.digital/manuals/programming-languages/python/python.html#common-configuration)
-described on [The GDS Way](https://gds-way.cloudapps.digital).
+A configuration file for the [`flake8`][flake8] Python package that provides linting. This file is based on the
+[common configuration][gds-way-flake8] described on [The GDS Way][gds-way].
 
 ### `.gitignore`
 
 A `.gitignore` file to ignore certain files and folders from this Git repository. See the
-[GitHub Help pages](https://help.github.com/en/github/using-git/ignoring-files) for further information.
+[contributor guide][docs-updating-gitignore] for further information about modifying this file.
 
 ### `.pre-commit-config.yaml`
 
-A pre-commit hook configuration file. See the `pre-commit` package [documentation](https://pre-commit.com/) for further
-details.
+A pre-commit hook configuration file. See the [contributor guide][docs-pre-commit-hooks] for further details.
 {% if cookiecutter.create_secrets_file == "Yes" %}
 ### `.secrets`
 
 A file to store all secrets and credentials as environment variables. This is read-in by [`.envrc`](#envrc) using the
-[`direnv`](https://direnv.net/) shell extension, but is **not** tracked by Git.
+[`direnv`][direnv] shell extension, but is **not** tracked by Git.
 {% endif %}
 ### `.secrets.baseline`
 
-Optional baseline file for the [`detect-secrets`](https://github.com/Yelp/detect-secrets) package; this package detects
-secrets, and, in conjunction with `pre-commit`, prevents them from being committed to the repository. The baseline file
-flags secret-like data that the user deliberately wishes to commit the to repository.
+Baseline file for the [`detect-secrets`][detect-secrets] package; this package detects secrets, and, in conjunction
+with `pre-commit`, prevents them from being committed to the repository. The baseline file flags secret-like data that
+the user deliberately wishes to commit the to repository.
+
+### `CODE_OF_CONDUCT.md`
+
+The [Code of Conduct][code-of-conduct] for contributors to this project, including maintainers and `ukgovdatascience`
+organisation owners.
+
+### `CONTRIBUTING.md`
+
+The [contributing guidelines][contributing] for this project.
+
+### `LICENSE`
+
+The licence for this project. Unless stated otherwise, the codebase is released under the MIT License. This covers both
+the codebase and any sample code in the documentation. The documentation is © Crown copyright and available under the
+terms of the Open Government 3.0 licence.
 
 ### `Makefile`
 
 The `Makefile` contains a set of commands for the `make` utility. Run the `help` command for further information at the
 top-level of the Git repository.
 
-```
+```shell
 make help
 ```
 
+### `pytest.ini`
+
+A file containing configuration settings for the [`pytest`][pytest] Python package. To run tests within the
+[`tests`][docs-tests], and `{{ cookiecutter.repo_name }}/tests` folders, execute the following
```

Comment:
```suggestion
[`tests`](tests.md), and `{{ cookiecutter.repo_name }}/tests` folders, execute the following
```

[`docs-tests`] not defined, I assume is `docs/structure/tests.md`?

Reply:
Change made in commit [`212b899`](https://github.com/ukgovdatascience/govcookiecutter/pull/10/commits/212b899c7865d546ac3dcb936a39292687f4a850). Marking as **Resolved**.